authui/src/authflowv2/customSelect.ts

```diff
@@ -135,7 +135,8 @@ export class CustomSelectController extends Controller {
     if (!this.dropdownContainerTarget.classList.contains("hidden")) return;
 
     this.dropdownContainerTarget.classList.remove("hidden");
-    this.triggerTarget.setAttribute("aria-expanded", "true");
+    this.triggerTarget.classList.add("select__trigger--expanded");
```

Comment:
You remove `this.triggerTarget.setAttribute("aria-expanded", "true");` but in line 155, its counterpart is kept. What is the intention?

Reply:
Should be a mistake